front/pages/terms.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <layout>
+    <div class="min-vh100-fh">
+      TODO Terms page
+    </div>
+
+    <main-footer />
+  </layout>
+</template>
+
+<script>
+import Layout from '@/components/layout/Layout'
+import MainFooter from '@/components/layout/MainFooter'
+
+export default {
+  components: {
```

Comment:
and all this pages are same, so base component like Static Page could be used on all of them instead of copy paste repeat.

Reply:
I mentioned that about footer see #282 , footer should be in the general layout as it most likely will be present on any page, do you suggest to create a separate layout just for static pages where the difference is just footer inclusion?

Reply:
Im talkling about creating new component which include all this page code and use slot instead of  TODO with text, that's it

Reply:
Ok